buku

```diff
@@ -4069,16 +4083,8 @@ def is_bad_url(url):
         True if URL is malformed, False otherwise.
     """
 
-    # Get the netloc token
-    try:
-        netloc = parse_url(url).netloc
-        if not netloc:
-            # Try of prepend '//' and get netloc
-            netloc = parse_url('//' + url).netloc
```

Comment:
Incidentally browsers seem to treat URLs starting with `//` as `file:` URLs

Also, fun fact: this doesn't really do anything when used with `parse_url()` :sweat_smile: